DockDoor/Components/Marquee.swift

```diff
@@ -63,8 +67,8 @@ struct TheMarquee<C: View>: View {
         .measure($containerSize)
         .compositingGroup()
         .opacity(measured ? 1 : 0)
-        .onChange(of: containerSize) { _, _ in startAnimation() }
```

Comment:
Let's keep these as underscores, the variable is not used.

Reply:
I just removed the second parameter as it is not supported in macOS 13, i did not change it to a named parameter